pcli/tests/test_integration.rs

```diff
@@ -0,0 +1,118 @@
+//! Basic integration testing of `pcli` versus a target testnet.
+//!
+//! These tests are marked with `#[ignore]`, but can be run with:
+//! `cargo test --package pcli -- --ignored`
+//!
+//! Tests against `testnet-preview.penumbra.zone` by default, override with
+//! environmental variable `PENUMBRA_NODE`.
+//!
+//! Tests assume that the initial state of the test account is after genesis,
+//! where no tokens have been delegated, and the address with index 0
+//! was distributed 1cube.
```

Comment:
I think we'll need to allocate penumbra tokens to the test account, because otherwise we have no way to test (un)delegations...